README.md

```diff
@@ -0,0 +1,15 @@
+# Score Sheet Bot
+
+## Development
+
+### Python Setup
+
+1. Optional: Create `venv`, and activate via `venv/Scripts/activate`
+1. Install dependencies with `pipenv install`
+
+### Discord Bot Setup
+
+1. Create an application at https://discord.com/developers/applications
+1. Under the app, create a Bot
+1. Copy `.envexample` to `.env`, and add the token from the Bot page
+1. Invite your bot to your test server via `https://discord.com/api/oauth2/authorize?client_id={CLIENT_ID}&permissions=519232&scope=bot`, where the client id is found in your General Information page
```

Comment:
1. Add a section on running the bot (Just run main.py)
2. On the first run, you will be prompted to login to your google account and will need to give the bot access to view documents. This will create a token.pickle file that you need to use otherwise it will not be able to properly parse data from the spreadsheet.
3. The bot (in its current configuration) does not work not on the SCS server, I will need to change things to work in a testing discord.